index.bs

```diff
@@ -1,14 +1,160 @@
+
+## Pause data-burst ## {#pause-burst}
+
+The pause data-bursts for IAMF are defined in Table 2
+
+<center><img src="table-2-repetition-period-of-pause-data-burst.png" width="600"></center>
+<center><b>Table 2 – Repetition period of pause data-burst</b></center>
+
+If regular audio data-bursts are not being transmitted due to, for example, a pause condition, it is recommended to use pause data-bursts to fill such stream gaps. The repetition period of the pause data-bursts should be selected according to Table 2. If other repetition periods are necessary to precisely fill the stream gap length, or to meet the requirement on audio data-burst spacing (see IEC 61937-1), pause data-bursts may have other lengths.
+
+When a stream gap in an audio stream is filled by a sequence of pause data-bursts, the Pa of the first pause data-burst shall occur after exactly that number of IEC 60958 frames as indicated by the IAMF audio data-frame length in conjunction with the data-type-dependent information from Table 3. It is recommended that the sequence of pause data-bursts that fills the stream gap should continue from this point until the Pa of the first audio data-burst that follows the stream gap, or as close as possible to that point, considering the specific IEC 60958 frame length of the pause data-burst with respect to the IAMF audio data frame length. The repetition-period-length parameter contained in the pause data-burst is intended to be interpreted by the receiver as an indication of the number of decoded PCM samples that are missing (due to the resulting audio gap).
+
+## Audio data-burst ## {#audio-burst}
+
+### The IAMF data ### {#iamf-data}
+An IAMF bit stream consists of a sequence of IAMF audio data frames. The IAMF data-burst is headed with a burst-preamble, followed by the burst-payload. The structure of the IAMF data-burst is shown in Figure 1. The extended data-type Pe bits 0 to 15 of an IAMF data-burst is 0x0003h, and the data-type Pc bits 8 to 9 is 0. When IAMF data is being transmitted, the transmission device shall ensure that both the extended data-type Pe bits 0 to 15 and data-type Pc bits 8 to 9 values are set correctly. Additionally, the receiving device shall utilize both the extended data-type Pe bits 0 to 15 and data-type Pc bits 8 to 9 values to ensure that the content of the data-burst is correctly identified as IAMF.
+
+<center><img src="figure-1-IAMF-data-burst-structure.png" width="800"></center>
+<center><b>Figure 1. IAMF data-burst structure</b></center>
+
+The IEC 60958 frame rate for data-type IAMF shall be equal to the audio sample rate.  The repetition period of data-bursts in IEC 60958 frames shall be determined from the IAMF audio data frame length in accordance with Table 1 (Pc bits 10-11). The maximum data-burst payload size in bits is determined from 2 x 16 x Rp – (Paf + Bs), where
```

Comment:
would be good to be consistent about the multiply operator (x is used here, * is used below)